### DISCUSSION TITLE
Welcome to CALI-Author-Viewer-5 Discussions!

### DISCUSSION THREAD

**Discussion by sgoshorn** [CCALI/CALI-Author-Viewer-5#173] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emasters** [comment]
This is a good idea.

On Mon, Apr 8, 2024 at 11:36 AM Sam Goshorn ***@***.***>
wrote:

> 👋 Welcome!
>
> We’re using Discussions as a place to connect with other members of our
> community. We hope that you:
>
>    - Ask questions you’re wondering about.
>    - Share ideas.
>    - Engage with other community members.
>    - Welcome others and are open-minded. Remember that this is a
>    community we
>    build together 💪.
>
> To get started, comment below with an introduction of yourself and tell us
> about what you do with this community.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CCALI/CALI-Author-Viewer-5/discussions/173>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFU654KU27NLQXFAXOZX7TY4K2QZAVCNFSM6AAAAABF46S6ZWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGQ3TQOJZGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Elmer R. Masters
Director of Technology
Center for Computer-Assisted Legal Instruction
***@***.***    773-332-7508

**Reply by sgoshorn** [reply]
I concur!
